source/core/src/main/com/csse3200/game/entities/factories/ItemFactory.java

```diff
@@ -46,6 +50,33 @@ public static Entity createHealthPotion(Entity target) {
         return createItem(target, new HealingPotion(1));
     }
 
+    /**
+     * Creates a defense potion entity that interacts with the specified target.
+     *
+     * @param target The entity the defense potion will interact with
+     * @return The created health potion entity.
+     */
+    public static Entity createDefensePotion(Entity target) {
+        GameTime gameTime = new GameTime();
+        return createItem(target, new DefensePotion(1, gameTime));
+    }
+
+    public static Entity createSpeedPotion(Entity target) {
+        GameTime gameTime = new GameTime();
+        return createItem(target, new SpeedPotion(1, gameTime));
+    }
+
+    /**
+     * Creates an attack potion entity that interacts with the specified target.
+     *
+     * @param target The entity the attack potion will interact with
+     * @return The created attack potion entity
+     */
```

Comment:
These functions are never actually used - they should be added to the handleItems function in ForestGameArea so this can be functionally tested.